### PR TITLE
Add acknowledgement

### PR DESCRIPTION
**Pull request by glyn**
@cburgmer do you think the reference should include an organisation?

### CODE REVIEW

**Review by gregsdennis — APPROVED** [review]

**Review by cabo — APPROVED** [review]
Thank you for catching this!

File: draft-ietf-jsonpath-base.md [hunk 1]
```diff
@@ -77,16 +77,17 @@ informative:
     target: https://cburgmer.github.io/json-path-comparison/
     title: JSONPath Comparison
     author:
-      name: Christoph Burgmer
+    - name: Christoph Burgmer
```
Comment (cabo) [inline draft-ietf-jsonpath-base.md:80]: That doesn't look right.

Comment (cabo) [reply draft-ietf-jsonpath-base.md:80]: Organizations are not shown in references, unless they *are* the author (you now have two authors, one of which is the person, and one is the organization, which is certainly broken).

File: draft-ietf-jsonpath-base.md [hunk 2]
```diff
@@ -73,15 +73,21 @@ contributor:
   uri: https://github.com/gregsdennis
 
 informative:
+  COMPARISON:
+    target: https://cburgmer.github.io/json-path-comparison/
+    title: JSONPath Comparison
+    author:
+    - name: Christoph Burgmer
+    - org: Thoughtworks
```
[the remark this answers is not in the document]
Comment (cabo) [reply draft-ietf-jsonpath-base.md:81]: https://github.com/ietf-wg-jsonpath/draft-ietf-jsonpath-base/pull/501#discussion_r1288406582

File: draft-ietf-jsonpath-base.md (hunk 1 above)
Comment (glyn) [reply draft-ietf-jsonpath-base.md]: Ok, I've switched to the same format as JSONPath-original (which I've also reverted). I think it's more important that the authors get the credit than their employers.

@cburgmer: please note with my apologies for thinking I could include the organisation in the rendered output.

**Pull request merged**